Ontology/Capability/Sensor/StatusSensor/DamperStatusSensor.json

```diff
@@ -1,6 +1,15 @@
 {
   "@id": "dtmi:digitaltwins:rec_3_3:core:DamperStatusSensor;1",
   "@type": "Interface",
-  "extends": "dtmi:digitaltwins:rec_3_3:core:StatusSensor;1",
+  "displayName": {
+    "en": "Damper Status Sensor"
+  },
+  "extends" : [
+    "dtmi:digitaltwins:rec_3_3:core:StatusSensor;1",
+    "dtmi:digitaltwins:rec_3_3:core:DamperStatusSensor;1"
```

Comment:
DamperStatusSensor should not extend itself.

Reply:
Fixed